src/bmconfigparser.py

```diff
@@ -7,7 +7,6 @@
 import os
```

Comment:
I have seen only copyfile method from shutil.

@g1itch can you import the copyfile method  from shutil
```
from shutil import copyfile
```
instead of shutil library.